### PR TITLE
fix(fx-2730): custom price filter q/a

### PR DESCRIPTION
**Pull request by dzucconi**
Closes: [FX-2735](https://artsyproduct.atlassian.net/browse/FX-2735)

### CODE REVIEW

File: src/v2/Components/v2/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx [hunk 1]
```diff
@@ -12,6 +13,23 @@ import {
 } from "@artsy/palette"
 import { useArtworkFilterContext } from "../ArtworkFilterContext"
 import { OptionText } from "./OptionText"
+import styled from "styled-components"
+import { Media } from "v2/Utils/Responsive"
+
+// Disables arrows in numeric inputs
```
Comment (mzikherman) [inline src/v2/Components/v2/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx:19]: interesting!

File: src/v2/Components/v2/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx [hunk 2]
```diff
@@ -71,69 +93,105 @@ export const PriceRangeFilter: React.FC = () => {
...
-          </Button>
-        </>
-      )}
-    </Toggle>
+    <>
+      <Toggle label="Price" expanded>
+        {mode === "done" && (
+          <Media lessThan="sm">
+            <Message variant="info" my={2}>
+              Price set; select apply see full results
```
Comment (mzikherman) [inline src/v2/Components/v2/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx:115]: typo, should it be "select apply **to** see full results"?

**Comment by mzikherman** [comment]
LGTM! Conflix but merge whenever.

File: src/v2/Components/v2/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx (hunk 2 above)
Comment (dzucconi) [reply src/v2/Components/v2/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx:115]: Good catch (copied from Figma!)

**Pull request merged**